platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformComponentBuilder.java

```diff
@@ -464,4 +467,36 @@ public EventCreationManager buildEventCreationManager() {
         }
         return eventCreationManager;
     }
+
+    /**
+     * Provide an in-order linker in place of the platform's default in-order linker.
+     *
+     * @param inOrderLinker the in-order linker to use
+     * @return this builder
+     */
+    @NonNull
+    public PlatformComponentBuilder withInOrderLinker(@NonNull final InOrderLinker inOrderLinker) {
```

Comment:
unused method.

Reply:
This is used. See `PlatformWiringTests.java`.